travis/travis_install_nightly

```diff
@@ -32,6 +32,11 @@ ln -s `which nodejs` $HOME/maintainer-quality-tools/travis/node
 npm install -g less less-plugin-clean-css
 
 
+# Update PhantomJS (v10 compat)
+npm install --prefix ${TRAVIS_BUILD_DIR} phantomjs-prebuilt@2.1.14
```

Comment:
I can do `@latest`, but that's about the only wild-type version from what I understand.